ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-service-insertion.j2

```diff
@@ -7,4 +7,48 @@
 {% if router_service_insertion.enabled is arista.avd.defined(true) %}
 
 Router service-insertion is enabled.
+{%     if router_service_insertion.connections is arista.avd.defined %}
+
+## Network Services Information
+{%         for connection in router_service_insertion.connections | arista.avd.natural_sort('name') %}
+{%             if connection.name is arista.avd.defined and connection.interface is arista.avd.defined %}
```

Comment:
`name` is always defined as this is the primary key (once you make it in the schema)

Reply:
Removed name from condition.